client/src/components/TextInputComponent/TextInput.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./TextInput.css";
+
+const TextInput = props => (
+  <>
+    <label className="labelNameClass">{props.labelName}</label>
+    <br />
```

Comment:
Can you achieve this spacing with CSS, rather than the `br` tag?

Reply:
changed the styling display: block! since the label is inline